Documented Windows Forms compatibility changes

rpetrusha opened the pull request:
## Documented Windows Forms compatibility changes

@PriyaPurkayastha

Commit:
Documented Windows Forms compatibility changs

mairaw commented on docs/core/compatibility/3.0.9-3.0.md:
```diff
@@ -0,0 +1,24 @@
+
+[!INCLUDE[Switch.System.Windows.Forms.AllowUpdateChildControlIndexForTabControls compatibility switch not supported](~/includes/core-changes/windowsforms/deprecate-allowupdatechildcontrolindexfortabcontrols.md)]
+
+[!INCLUDE[Switch.System.Windows.Forms.EnableVisualStyleValidation compatibility switch not supported](~/includes/core-changes/windowsforms/deprecate-enablevisualstylevalidation.md)]
```
wondering if it would look nice (or not) to have an horizontal rule in between changes. Right now it's hard to see where they begin and end.

```suggestion
***

[!INCLUDE[Switch.System.Windows.Forms.EnableVisualStyleValidation compatibility switch not supported](~/includes/core-changes/windowsforms/deprecate-enablevisualstylevalidation.md)]
```

mairaw reviewed (approved):
LGTM. Left a few comments for you to consider.

Commit:
Apply suggestions from code review

Thanks, @mairaw, for both the corrections and the suggested improvements! Your feedback is great!

Co-Authored-By: Maira Wenzel <mairaw@microsoft.com>

Commit:
Added separating lines

Commit:
Removed fencing from switch in headers
